images/bootstrap/create_bazel_cache_rcs.sh

```diff
@@ -24,9 +24,9 @@ package_to_version () {
 # look up a binary with which and return the debian package it belongs to
 command_to_package () {
     local binary_path
-    binary_path=$(readlink -f "$(which "$1")")
-    # `dpkg -S $package` spits out lines with the format: "package: file"
-    dpkg -S "$1" | grep "${binary_path}" | cut -d':' -f1
+    binary_path=$(readlink -f "$(command -v "$1")")
```

Comment:
http://pubs.opengroup.org/onlinepubs/009696899/utilities/command.html

Reply:
for this we don't really, but I at least didn't know about `command -v` and shellcheck prefers `command -v` for portability and it accomplishes the same task. fixing this makes this file pass shellcheck with no errors.

Reply:
`planter/planter.sh` is the only intentionally posix sh script, because it's meant to run in user shells which could be anything and it doesn't really need bash features.